src/features/heroSection/components/SvgFiles/HeroSectionRightSvg.jsx

```diff
@@ -75,7 +78,7 @@ const HeroSectionRightSvg = ({ imgfile }) => {
           width="768"
           height="512"
           // xlinkHref={`img/${imgfile}`}
-          xlinkHref="img/tree.png"
+          xlinkHref={image2}
```

Comment:
why u hardcoding the image and not send as property , this component will only show this specific image ? if not then it should get the image link as prop so it can be resuable